changelog.d/20241212_121806_PL-133165-nix-2.3-backport_scriv.md

```diff
@@ -0,0 +1,9 @@
+### Impact
+
+- Restart of `nix-daemon.service` due to default Nix being changed.
```

Comment:
this can be removed, there is no customer application impact.

Reply:
Done.